crates/ruff_python_parser/resources/invalid/statements/match/invalid_class_pattern.py

```diff
@@ -0,0 +1,17 @@
+# Invalid keyword pattern in class argument
+match subject:
+    case Foo(x as y = 1):
+        pass
+    case Foo(x | y = 1):
+        pass
+    case Foo([x, y] = 1):
+        pass
+    case Foo({False: 0} = 1):
+        pass
+    case Foo(1=1):
+        pass
+    case Foo(Bar()=1):
+        pass
+    # Positional pattern cannot follow keyword pattern
+    # case Foo(x, y=1, z):
+    #     pass
```

Comment:
Is this intentionally commented out?

Reply:
Yes. So, positional pattern cannot follow keyword pattern in a class pattern (`case Foo(x=1, y): ...` is invalid) but this means the pre-order assert in the invalid code fails.

I'll probably need to have a mathod on `PatternArguments` similar to how's one for `Arguments` (`arguments_source_order`) to return an argument in the order defined in source.

I need to think about this because it's only required while testing.